### PR TITLE
3958 redirect

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/presenter/actions/getShouldRedirectToSigningAction.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  });
+
+  it('should call the no path for court issued documents that have a Notice event code', async () => {
+    await runAction(getShouldRedirectToSigningAction, {
+      modules: {
+        presenter,
+      },
+      props: {
+        documentId: '123',
+        eventCode: 'O',
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/getShouldRedirectToSigningAction.test.js]: I think this should be eventCode 'NOT'

File: web-client/src/presenter/actions/getShouldRedirectToSigningAction.test.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  });
+
+  it('should redirect to the document detail page for court issued documents that have a Notice event code', async () => {
+    const result = await runAction(getShouldRedirectToSigningAction, {
+      modules: {
+        presenter,
+      },
+      props: {
+        documentId: '123',
+        eventCode: 'O',
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/getShouldRedirectToSigningAction.test.js]: This should also be eventCode 'NOT'

File: web-client/src/presenter/actions/getShouldRedirectToSigningAction.test.js (hunk 1 above)
Comment (matthopson) [reply web-client/src/presenter/actions/getShouldRedirectToSigningAction.test.js]: Well, that's embarrassing. Fixed.

File: web-client/src/presenter/actions/getShouldRedirectToSigningAction.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { state } from 'cerebral';
+
+export const getShouldRedirectToSigningAction = ({
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/getShouldRedirectToSigningAction.js:14]: Let's drop some jsDocs in here before we forget.

**Review by kkoskelin — APPROVED** [review]

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**